examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,362 @@
+
+# %%
+# Visualise the head vector
+# ---------------------------
+# To visually check our computation of the head vector, it is easier to select
+# a subset of the data. We can focus on the trajectory of the head when the
+# mouse is within a small rectangular area, and within a certain time window.
+
+# area of interest
+xmin, ymin = 600, 665  # pixels
+x_delta, y_delta = 125, 100  # pixels
+
+# time window
+time_window = range(1650, 1670)  # frames
+
+
+# %%
+# For that subset of the data, we now plot the head vector.
+
+fig, ax = plt.subplots(1, 1)
+mouse_name = ds.individuals.values[0]
+
+# plot midpoint between the ears, and color based on time
+sc = ax.scatter(
+    midpoint_ears.sel(individuals=mouse_name, space="x", time=time_window),
+    midpoint_ears.sel(individuals=mouse_name, space="y", time=time_window),
+    s=50,
+    c=midpoint_ears.time[time_window],
+    cmap="viridis",
+    marker="*",
+)
+
+# plot snout, and color based on time
+sc = ax.scatter(
+    position.sel(
+        individuals=mouse_name, space="x", time=time_window, keypoints="snout"
+    ),
+    position.sel(
+        individuals=mouse_name, space="y", time=time_window, keypoints="snout"
+    ),
+    s=50,
+    c=position.time[time_window],
+    cmap="viridis",
+    marker="o",
+)
+
+# plot the computed head vector
+ax.quiver(
+    midpoint_ears.sel(individuals=mouse_name, space="x", time=time_window),
+    midpoint_ears.sel(individuals=mouse_name, space="y", time=time_window),
+    head_vector.sel(individuals=mouse_name, space="x", time=time_window),
+    head_vector.sel(individuals=mouse_name, space="y", time=time_window),
+    angles="xy",
+    scale=1,
+    scale_units="xy",
+    headwidth=7,
+    headlength=9,
+    headaxislength=9,
+    color="gray",
+)
+
+ax.axis("equal")
+ax.set_xlim(xmin, xmin + x_delta)
+ax.set_ylim(ymin, ymin + y_delta)
+ax.set_xlabel("x (pixels)")
+ax.set_ylabel("y (pixels)")
+ax.set_title(f"Zoomed in to check ({mouse_name})")
+ax.invert_yaxis()
+fig.colorbar(sc, ax=ax, label=f"time ({ds.attrs['time_unit']})")
+
+ax.legend(
+    [
+        "midpoint_ears",
+        "snout",
+        "head_vector",
+    ],
+    loc="best",
+)
+
+fig.show()
+
+# %%
+# From the plot we can confirm the head vector goes from the midpoint between
+# the ears to the snout, as we defined it.
+
+
+# %%
+# Express the head vector in polar coordinates
+# -------------------------------------------------------------
+# A convenient way to inspect the orientation of a vector in 2D is by
+# expressing it in polar coordinates. We can do this with the vector function
+# ``cart2pol``:
+head_vector_polar = cart2pol(head_vector)
+
+print(head_vector_polar)
+
+# %%
+# Notice how the resulting array has a ``space_pol`` dimension with two
+# coordinates: ``rho`` and ``phi``. These are the polar coordinates of the
+# head vector.
+#
+# The coordinate ``rho`` is the norm (length) of the vector. In our case, the
```

Comment:
maybe also mention the term "magnitude". It's all the same, but some people maybe more familiar with one of these terms.